hacks/reconcile-trigger.sh

```diff
@@ -0,0 +1 @@
+kubectl -n garden-local annotate shoot local "gardener.cloud/operation=reconcile" --overwrite
```

Comment:
Missing copyright notice.

Reply:
too simple - I'm not adding License notice to signle commad kubectl scripts :)